environment.yaml

```diff
@@ -3,9 +3,10 @@ channels:
     - conda-forge
 dependencies:
     - black=23.1.0
-    - python=3.10.9
+    - cookiecutter=2.1.1
     - pre-commit=3.0.4
     - pylint=2.16.1
     - pytest=7.2.1
-    - cookiecutter=2.1.1
     - pytest=7.2.1
+    - python=3.10.9
+    - ruff=0.1.4
```

Comment:
I don't think that we need ruff in the top-level environment, as we're only using one linter for the template repository itself. Let's stick with Pylint for now, so I guess you can revert all changes to that file.

Reply:
Fixed with rebase onto main